soccer/motion/TrapezoidalMotionTest.cpp

```diff
@@ -3,17 +3,24 @@
 #include <Geometry2d/Point.hpp>
 #include <motion/TrapezoidalMotion.hpp>
 #include <math.h>
+#include <iostream>
 
+using namespace std;
 
 bool trapezoid1(float t, float &posOut, float &speedOut) {
```

Comment:
Ahh I gotcha.  These test are kind of a mess though, can you clean them up a bit and explain what each one tests?  Also, naming convention for google test is `TEST(ClassName, WhatsBeingTested)`.